Non-Interactive-Protocol.md

```diff
@@ -0,0 +1,173 @@
+# Non-interactive Onchain Protocol
+
+## Abstract
+
+DLC allows two parties to conduct stateful bitcoin contracts by giving each
+of the parties *cross-signed range of CET transactions*. These CET transactions
+are all spending the same output from a funding transaction and thus only one of
+the alternative can confirm. A CET is finalized according to the outcome signed
+by the previously selected oracle.
+
+As the contract is established once funding transaction signatures have been exchanged,
+from then a party should assume counterparty non-cooperation and ensure either
+confirmation of the funding or double-spend of its collatral inputs to avoid exploitation.
+
+In the optimistic way, once the oracle signature is released, parties interact
+off-chain to settle the DLC. They generate a *closing_tranaction* and publish it on the
+blockchain.
+
+In the pessimitic way, a party interact with the chain to unilaterally settle the DLC.
+They are two scenarios to consider:
+
+1.  The oracle releases the outcome signature but the DLC counterparty goes wrong,
+either going offline or maliciously behaving. The party must broadcast the DLC
+transaction paired with the outcome.
+
+2. The oracle doesn't release the outcome signature, independently of counterparty
+behavior, the party must broadcast the refund transaction after its timelock expiration.
+
+# Table of Contents
+	* [General Requirements](#general-requirements)
+	* [Funding Phase](#funding-phase)
+	* [Execution Phase](#execution-phase)
+	* [Fee-Bumping Policy](#fee-bumping-policy)
+	* [Reorgs] (#reorgs)
+
+# General Requirements
+
+A DLC client should have censorship-resistant access to the blockchain, use a
+local fee-estimator and ensure it's well-connected to the tx-relay network.
+
+We recommend to not react on any mempool event as otherwise it would be a source of
+cross-layer mapping.
+
+We recommend to not sign local transactions until their broadcast is required
+to avoid their misuage in case of site compromise, assuming an external signer.
```

Comment:
Again with adaptor version there is no really notion of local transaction. You need to generate the adaptor signature for the CETs which are the same for both parties. But maybe you can mention that they should not store them?

Reply:
Right, I remove the local transaction mention. They can either not store the signature or avoid to produce them at all until it's needed. That's a recommendation not mandatory behavior as it depends on your signing setup.

I use the word local few times to mean this holder-of-the-CET-and-not-the-counterparty e.g in the context of signature. Should I upgrade local -> holder ?

Reply:
No I think local is fine if you're talking about signature.